app/feature/odyssey/src/main/kotlin/com/hedvig/android/feature/odyssey/step/selectcontract/SelectContractViewModel.kt

```diff
@@ -0,0 +1,90 @@
+package com.hedvig.android.feature.odyssey.step.selectcontract
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.hedvig.android.data.claimflow.ClaimFlowRepository
+import com.hedvig.android.data.claimflow.ClaimFlowStep
+import com.hedvig.android.data.claimflow.LocalContractContractOption
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+
+internal class SelectContractViewModel(
+  initialSelectedContract: String?,
```

Comment:
Aha I see what this is here then. For the initial selection.
I don't think we're ever gonna decide on the initial selection from this parameter, in fact I see it's just `null` all the time where we instantiate this ViewModel.

Can we instead not have this parameter at all and do the "selecting the first contract by default" logic just in here instead?